base/sysimg.jl

```diff
@@ -180,6 +180,27 @@ big(n::Integer) = convert(BigInt,n)
 big(x::FloatingPoint) = convert(BigFloat,x)
 big(q::Rational) = big(num(q))//big(den(q))
 
+const _fact_table128 =
```

Comment:
I think it will be fine if I move the `binomial` and `factorial` from `gmp.jl` and `mpfr.jl` into `combinatorics.jl`.